dark/index.html

```diff
@@ -147,7 +147,7 @@ <h1>
       </div>
       <div id="tooltip">
         <span id="tooltiptext">Twitter</span>
-        <a href="https://twitter.com/Yoshitha2008" target="_blank "><img src="https://img.icons8.com/?size=48&id=YfCbGWCWcuar&format=png&color=ffffff" alt="Twitter Icon" /></a>
+        <a href=" https://x.com/Yoshitha_R" target="_blank "><img src="https://img.icons8.com/?size=48&id=YfCbGWCWcuar&format=png&color=ffffff" alt="Twitter Icon" /></a>

```

Comment:
Remove the space here @S-125 
```suggestion
        <a href="https://x.com/Yoshitha_R" target="_blank "><img src="https://img.icons8.com/?size=48&id=YfCbGWCWcuar&format=png&color=ffffff" alt="Twitter Icon" /></a>

```